internal/privi/repo.go

```diff
@@ -89,27 +124,74 @@ func NewSQLiteRepo(db *sql.DB) (*sqliteRepo, error) {
 	}, nil
 }
 
+// TODO: does this need to recurse more than one layer?
+func getBlobRefsFromSchema(sch *lexicon.SchemaFile, rec record) ([]syntax.CID, error) {
```

Comment:
i think we should be able to just do a string/regex match to determine this, right? then we can push it into the sql query